sale_product_pack/models/sale_order_line.py

```diff
@@ -78,8 +78,12 @@ def create(self, vals_list):
         new_vals = []
         res = self.browse()
         for elem in vals_list:
-            product = self.env["product.product"].browse(elem["product_id"])
-            if product.pack_ok and product.pack_type == "detailed":
+            product = (
+                self.env["product.product"].browse(elem["product_id"])
+                if "product_id" in elem
+                else None
+            )
```

Comment:
I think that could be improved like this, what do you think of this?
```suggestion
            product = self.env["product.product"].browse(elem.get("product_id"))
```

Reply:
Hi @augusto-weiss, i forgot to mention you during my review. Thanks for the bug fix, maybe you could review the initial pull request, maybe i made others mistakes, i hope not.